src/mad_spark_alt/core/retry.py

```
@@ -358,14 +358,19 @@ async def _make_request() -> Any:
 
                 return response_data
 
-        except aiohttp.ClientTimeout:  # type: ignore
-            raise LLMTimeoutError("Request timed out")
+        except asyncio.TimeoutError as e:
+            raise LLMTimeoutError("Request timed out") from e
```

Ruff (0.11.9)

Avoid specifying long messages outside the exception class

(TRY003)